text-ui-test/EXPECTED.TXT

```diff
@@ -122,19 +126,19 @@ Hello from...
     ____________________________________________________________
      Got it. I've added this task:
        [T][N] iP meeting (p:0)
-     Now you have 1 task(s) in the list.
+     Now you have 1 items in the list.
```

Comment:
Hmm why is this changed, I thought @MuhammadHoze updated it and this is now the intended behaviour.

Reply:
Oops I did not notice it. I just copied from ACTUAL.TXT to EXPECTED.TXT. I think I might have accidentally changed it when resolving merge conflicts. I will change it back.